be/src/exec/pipeline/exchange/local_exchange.h

```diff
@@ -64,7 +64,7 @@ class PartitionExchanger final : public LocalExchanger {
         LocalExchangeSourceOperatorFactory* _source;
         const bool _is_shuffle;
         // Compute per-row partition values.
-        const std::vector<ExprContext*>& _partition_expr_ctxs;
+        const std::vector<ExprContext*> _partition_expr_ctxs;
```

Comment:
Why `&` is removed, which will cause every `PartitionExchanger` copies `std::vector<ExprContext*>` from `LocalExchanger`?

Reply:
aggregator's decompose use a local variable, so copy needed here. And the cost of copy pointers is cheap

Reply:
Maybe we can add a member const std::vector<ExprContext*> _partition_expr_ctxs to LocalExchanger, and use reference to it in every PartitionExchanger